applications/nrf5340_audio/src/bluetooth/bt_content_control/content/bt_media_control.c

```diff
@@ -184,101 +236,172 @@ static void mcs_local_player_instance_cb(struct media_player *player, int err)
 	(void)media_proxy_ctrl_send_command(local_player, &cmd);
 }
 
-int ble_mcs_discover(struct bt_conn *conn)
+/**
+ * @brief	Send play or pause command to either local media player or peer
+ *
+ * param	cmd	Command to send
+ *
+ * @return	0 for success, error otherwise.
+ */
+static int play_pause(struct mpl_cmd *cmd)
+{
+	int ret = 0;
+
+	if (IS_ENABLED(CONFIG_BT_MCS)) {
+		ret = media_proxy_ctrl_send_command(local_player, cmd);
+	} else if (IS_ENABLED(CONFIG_BT_MCC)) {
+		/* Send cmd to all peers connected and has finished discovery */
+		for (uint8_t i = 0; i < CONFIG_BT_MAX_CONN; i++) {
+			if (mcc_peer[i].mcp_mcs_disc_status == FINISHED) {
+				if (mcc_peer[i].conn != NULL) {
+					ret = bt_mcc_send_cmd(mcc_peer[i].conn, cmd);
+					if (ret) {
+						LOG_WRN("Failed to send play/pause command: %d",
+							ret);
+						return ret;
+					}
+				}
+			} else {
+				LOG_WRN("MCS discovery has not finished: %d",
+					mcc_peer[i].mcp_mcs_disc_status);
+				return -EBUSY;
```

Comment:
Do we want to return if MCS discovery has not finished for one conn?

Reply:
Up for discussion, but I think it makes sense to wait until all are done to keep them in sync. It is only valid if the discovery has already been started

Reply:
Same as the comment above:
It is a bit non-optimal if we e.g. look at these two different scenarios:
1) mcc_peer[0] finished mcc_peer[1] not finished
2) mcc_peer[0] not finished mcc_peer[1] finished
The behavior will be different for these two scenarios

Reply:
As a note: This isn't gateway discovering the two headsets, this is on headsets. That means for all our current use cases there is only one mcc_peer. If there are two clients then that is two different gateways. The real problem here is that we don't know which of the mcc_peers to send the cmd to, since we don't get a pointer to the conn.

I've changed the logic now so that we first check if we have a conn pointer at all, then check if it is finished. If it is not finished it will continue searching and sending cmd to those that are finished. If it fails to send a cmd that is printed in a warn and stored in a temp that will be returned once finished